core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -563,7 +575,7 @@ private[spark] class AppStatusListener(
   /** Flush all live entities' data to the underlying store. */
   def flush(): Unit = {
     val now = System.nanoTime()
-    liveStages.values.foreach { stage =>
+    liveStages.values.asScala.foreach { stage =>
```

Comment:
Why add `asScala`?

Reply:
`liveStages` is not a Scala map anymore.

Reply:
Ah I missed that that class change was scala -> java